### PR TITLE
feature #5 playlist curation

### PR DESCRIPTION
**Pull request by NicholasMacasaet**
…mparisons to accomodate new ID attribute, added ChangePrivacy() and changeName() methods to editPlaylist use case interactor, modified all methods within editPlaylist to return a String notification of the action done. Created editPlaylistInputData to package information required for editing as well as its associated interface.

### CODE REVIEW

**Review by rafaelgacesa — DISMISSED** [review]
All the communal code changes look good, I trust that your changes to your use case are good. The documentation and style is all fine.

See the one comment - I don't think it makes a difference (hence my approval), but just wondering.

File: src/main/java/InputBoundary/editPlaylistInputBoundary.java [hunk 1]
```diff
@@ -1,4 +1,14 @@
 package InputBoundary;
 
 public interface editPlaylistInputBoundary {
+    default void removeSong(){
```
[the remark this answers is not in the document]
Comment (NicholasMacasaet) [reply src/main/java/InputBoundary/editPlaylistInputBoundary.java:4]: Intellij yelled at me and said interface abstact methods cant have body when I switched it from default to public

[the remark this answers is not in the document]
Comment (NicholasMacasaet) [reply src/main/java/InputBoundary/editPlaylistInputBoundary.java:4]: Ah, ok. I'll change that then

**Review by manya-mittal — APPROVED** [review]
These changes look great Nick! I agree that isPublic is not needed since we are not making playlists searchable. The new return messages for success and failure are also useful.

**Review by chrstinalin — APPROVED** [review]
Looks good now!

**Review by rafaelgacesa — APPROVED** [review]
Looks great! Thanks for fixing my code too. Doing this also removes all the bloater code smells, which is excellent.

**Pull request merged**